--- ../chromium-107.0.5304.87/base/process/memory_linux.cc
+++ ./base/process/memory_linux.cc
@@ -108,10 +108,8 @@
 bool UncheckedMalloc(size_t size, void** result) {
 #if BUILDFLAG(USE_ALLOCATOR_SHIM)
   *result = allocator_shim::UncheckedAlloc(size);
-#elif defined(MEMORY_TOOL_REPLACES_ALLOCATOR) || !defined(LIBC_GLIBC)
+#else
   *result = malloc(size);
-#elif defined(LIBC_GLIBC)
-  *result = __libc_malloc(size);
 #endif
   return *result != nullptr;
 }
@@ -119,10 +117,8 @@
 void UncheckedFree(void* ptr) {
 #if BUILDFLAG(USE_ALLOCATOR_SHIM)
   allocator_shim::UncheckedFree(ptr);
-#elif defined(MEMORY_TOOL_REPLACES_ALLOCATOR) || !defined(LIBC_GLIBC)
+#else
   free(ptr);
-#elif defined(LIBC_GLIBC)
-  __libc_free(ptr);
 #endif
 }
 
