--- ncspot.orig/Cargo.toml
+++ ncspot/Cargo.toml
@@ -69,6 +69,7 @@
 wayland_clipboard = ["wl-clipboard-rs"] # Support wayland clipboard - linux only
 share_selection = ["clipboard", "wl-clipboard-rs"] # Use the primary selection for sharing - linux only
 alsa_backend = ["librespot-playback/alsa-backend"]
+jackaudio_backend = ["librespot-playback/jackaudio-backend"]
 pulseaudio_backend = ["librespot-playback/pulseaudio-backend"]
 rodio_backend = ["librespot-playback/rodio-backend"]
 portaudio_backend = ["librespot-playback/portaudio-backend"]
